pkgs/development/tools/vcstool/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, python35Packages }:
+
+  python35Packages.buildPythonApplication rec {
+    name = "vcstool";
+    version = "0.1.31";
+    #no tests
+    doCheck = false;
+
+    propagatedBuildInputs = with python35Packages; [ pyyaml ];
+    src = fetchurl {
```

Comment:
fetchPypi

Reply:
I switched to `fetchFromGithub` in my patch to download the tests.